typescript/sdk/src/metadata/chainMetadataTypes.ts

```diff
@@ -111,6 +111,10 @@ export const ChainMetadataSchemaObject = z.object({
     .array(RpcUrlSchema)
     .describe('For cosmos chains only, a list of Rest API URLs')
     .optional(),
+  gRpcUrls: z
```

Comment:
don't feel strongly but I've tended to see this stylized as `grpc` instead of `gRpc` (e.g. https://github.com/cosmos/chain-registry/blob/master/injective/chain.json#L311)

Reply:
agreed, will fix